scripts/deploy.sh

```
@@ -0,0 +1,65 @@
+
+source $(dirname "$0")/environment.sh
+source $(dirname "$0")/health_check.sh
+source $(dirname "$0")/cleanup.sh
+
+LOG_DIR="/var/log/ukhsc-system-backend"
+mkdir -p $LOG_DIR
+
+echo "Deploying $1 to $TARGET_ENV..."
+determine_environment
+
+docker stop ukhsc-system-backend-api-$TARGET_ENV || true
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)